react/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -1,10 +1,24 @@
-import { isArray } from './utils';
+import isArray from './utils';
 
-export function min() {
-  isArray();
-  return Math.min();
+export function min(...args) {
+  if (args.length === 0) {
+    return undefined;
+  }
+  const items = isArray(args[0]) ? args[0] : args;
+  return Math.min(...items);
 }
 
-export function copy() {
+export function copy(object) {
+  return isArray(object) ? [...object] : { ...object };
+}
+
+export function reverseMerge(first, second) {
+  return [...second, ...first];
+}
 
+export function filterAttribs(object) {
```

Comment:
You can spread the arguments of the function, just to return the rest param that you need.